libopenage/util/compiler.h

```diff
@@ -8,25 +8,55 @@
  *
  * May contain some platform-dependent code.
  */
+ #include <ciso646>
 
 // pxd: from libcpp.string cimport string
 #include <string>
 
+/**
+ * DLL entry-point decorations.
+ */
+#if defined(_WIN32)
+	#if defined(libopenage_EXPORTS)
+		#define LIBOPENAGE_API __declspec(dllexport)
+	#else
+		#define LIBOPENAGE_API __declspec(dllimport)
+	#endif /* libopenage_EXPORTS */
+#else
+	#define LIBOPENAGE_API
+#endif
+
+#if defined(_MSC_VER)
+#ifndef HAVE_SSIZE_T
+typedef int ssize_t;
```

Comment:
Hm I think that is not correct, as `int` is not always the size of `size_t`.

http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_types.h.htmlhttp://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_types.h.html

> The type `ssize_t` shall be capable of storing values at least in the range `[-1, {SSIZE_MAX}]`.

Strictly speaking, we've used `ssize_t` wrong in some places in codebase, so we should fix that someday :)

On my machine `SSIZE_MAX` is defined as `LONG_MAX`, which is `2 ** 63 - 1`, and bigger than `int` (`2 ** 31  - 1`).